src/main/java/racingcar/model/Racer.java

```diff
@@ -0,0 +1,76 @@
+package racingcar.model;
+
+import racingcar.constant.message.Message;
+import racingcar.dto.Result;
+import racingcar.model.car.Car;
+import racingcar.model.car.OrderByPosition;
+import racingcar.model.car.OrderStrategy;
+import racingcar.service.Accelerator;
+import racingcar.validation.Validator;
+import racingcar.validation.ValidatorFactory;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Racer {
+
+    private final List<Car> racer;
+    private final OrderStrategy orderStrategy;
+
+    private Racer(String value, OrderStrategy orderStrategy) {
+        validate(value);
+        this.racer = Arrays.stream(value.split(Message.NAME_SEPARATOR))
+                .map(String::trim)
+                .map(Car::ofStartingPoint)
+                .toList();
+        this.orderStrategy = orderStrategy;
+    }
```

Comment:
```suggestion
    private Racer(String value, OrderStrategy orderStrategy) {
        validate(value);
        this.racer = convertRacer(value);
        this.orderStrategy = orderStrategy;
    }

    private static List<Car> convertRacer(String value) {
        return Arrays.stream(value.split(Message.NAME_SEPARATOR))
                .map(String::trim)
                .map(Car::ofStartingPoint)
                .toList();
    }
```
메소드를 분리하면 가독성이 더 좋아질 것 같습니다!